README.md

```diff
@@ -1,7 +1,18 @@
 # rancher/hardened-build-base
 
+This repository holds the Dockerfiles and builds scripts for [rancher/hardened-build-base](https://hub.docker.com/r/rancher/hardened-build-base) Docker images. The `x86_64` image contains a Go compiler with FIPS 140-2 compliant crypto module, [GoBoring](https://github.com/golang/go/tree/dev.boringcrypto/misc/boring), used for [compiling rke2 components](https://docs.rke2.io/security/fips_support/#fips-support-in-cluster-components).
+
+Supported architectures
+
+- [x86_64/amd64](Dockerfile.amd64)
+- [s390x](Dockerfile.s390x)
+
 ## Build
 
-```sh
-TAG=v1.14.2 make
-```
\ No newline at end of file
+```shell
```

Comment:
What effect did this have on the output?

Reply:
The output varies depending on the markdown renderer, but in most cases it won't do anything to be honest. I did it mostly to appease my IDE's markdown linter, and I can change it to `sh` and keep it that way for future contributions.

Reply:
Let's keep consistent for now.

Reply:
Sure thing, I just changed it to `sh`